test/integ/dataplaneapi/conftest.py

```diff
@@ -0,0 +1,178 @@
+    s3.upload_file(testing_env_variables['MEDIA_PATH'] + testing_env_variables['SAMPLE_IMAGE'], stack_resources['DataplaneBucket'], 'upload/' + testing_env_variables['SAMPLE_IMAGE'])
+    s3.upload_file(testing_env_variables['MEDIA_PATH'] + testing_env_variables['SAMPLE_FACE_IMAGE'], stack_resources['DataplaneBucket'], 'upload/' + testing_env_variables['SAMPLE_FACE_IMAGE'])
+    s3.upload_file(testing_env_variables['MEDIA_PATH'] + testing_env_variables['SAMPLE_VIDEO'], stack_resources['DataplaneBucket'], 'upload/' + testing_env_variables['SAMPLE_VIDEO'])
+    # Wait for fixture to go out of scope:
+    yield upload_media
+
+
+@pytest.mark.usefixtures("upload_media")
+class API:
+    def __init__(self, stack_resources, testing_env_variables):
+        self.env_vars = testing_env_variables
+        self.stack_resources = stack_resources
+        self.auth = AWS4Auth(testing_env_variables['ACCESS_KEY'], testing_env_variables['SECRET_KEY'],
+                             testing_env_variables['REGION'], 'execute-api')
+
+        # aws_access_key = testing_env_variables['ACCESS_KEY'],
+        # aws_secret_access_key = testing_env_variables['SECRET_KEY'],
+        # aws_host = stack_resources['DataplaneApiEndpoint'].split('/')[0],
+        # aws_region = testing_env_variables['REGION'],
+        # aws_service = 'execute-api'
+
+    # dataplane methods
+
+    def create_asset(self):
+        headers = {"Content-Type": "application/json"}
+        body = {
+            "Input": {
+                "S3Bucket": self.stack_resources['DataplaneBucket'],
+                "S3Key": "upload/" + self.env_vars['SAMPLE_IMAGE']
+            }
+        }
+
+        print("POST /create")
+        create_asset_response = requests.post(self.stack_resources["DataplaneApiEndpoint"] + '/create', headers=headers,
+                                              json=body, verify=False, auth=self.auth)
+        return create_asset_response
+
+    def post_metadata(self, asset_id, metadata, paginate=False, end=False):
+        if paginate is True and end is True:
+            url = self.stack_resources["DataplaneApiEndpoint"] + 'metadata/' + asset_id + "?paginated=true&end=true"
+        elif paginate is True and end is False:
+            url = self.stack_resources["DataplaneApiEndpoint"] + 'metadata/' + asset_id + "?paginated=true"
+        else:
+            url = self.stack_resources["DataplaneApiEndpoint"] + 'metadata/' + asset_id
+
+        headers = {"Content-Type": "application/json"}
+        body = metadata
+        print("POST /metadata/{asset}".format(asset=asset_id))
+        nonpaginated_metadata_response = requests.post(url, headers=headers, json=body, verify=False, auth=self.auth)
+        return nonpaginated_metadata_response
+
+    # TODO: This test is currently broken. Seems to be a real issue with the API that needs looked into.
```

Comment:
Is there an issue open for this?  The issue should also cover enabling this test as part of the fix.

Reply:
Issue created.